Tweak vault description

wenottingham opened the pull request:
Vault isn't specifically for source control. Make description a little more generic and descriptive.

bcoca commented:
I would not remove the mention of source control, maybe rephrase:
"in encrypted files (which can be source controlled and shared) "

wenottingham committed:
Tweak vault description.

Vault isn't specifically for source control. Make description a little more generic and descriptive.

wenottingham commented:
Updated.
